community/procedure-compiler/processor/pom.xml

```diff
@@ -74,6 +73,18 @@
             <version>3.4.1</version>
             <scope>test</scope>
         </dependency>
+        <dependency>
+            <groupId>org.neo4j</groupId>
+            <artifactId>neo4j-security-enterprise</artifactId>
+            <version>${project.version}</version>
+            <scope>test</scope>
+        </dependency>
+        <dependency>
+            <groupId>org.neo4j</groupId>
+            <artifactId>neo4j-enterprise-kernel</artifactId>
+            <version>${project.version}</version>
+            <scope>test</scope>
+        </dependency>
```

Comment:
Community modules should not have dependencies on anything in enterprise. Tests that needs an enterprise runtime environment can be moved to enterprise. If you need to change runtime behaviour, there are usually a way with `Service.load` or similar.